### DISCUSSION TITLE
dyld[28218]: missing symbol called

### DISCUSSION THREAD

**Discussion by xchengood** [idaholab/moose#20709] [Q&A General] [answered]
Hi Moose experts,

I got one error named "dyld[28218]: missing symbol called". I am not sure it is caused by my test file or some other reasons. Any ideas to solve it?

**Comment by GiudGiud** [comment, marked as answer]
Hello

Is this post-reinstalling all your environment?
DId you `make clobberall` after re-installing?

Guillaume

**Reply by xchengood** [reply]
Hi @GiudGiud. Yes. I did `make clobberall` after re-installing. Some programs can be smoothly ran but my test file cannot. Actually, my test file can also be successfully ran before. I do not know why what happened.

**Reply by GiudGiud** [reply]
Weird.
Can you get us a full backtrace, following these instructions:

Also, what does `ldd <your executable>` return ?
Is this mac or linux btw?

Guillaume

**Reply by xchengood** [reply]
Hi @GiudGiud. Sorry. I cannot see those instructions you suggest using to check the backtrace. Could you write one more? Thank you.

**Reply by GiudGiud** [reply]
https://mooseframework.inl.gov/application_development/debugging.html
